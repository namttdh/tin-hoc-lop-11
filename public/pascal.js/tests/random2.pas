program Sample;
const
    max  = 10000;
var
    i    : Integer;
    rand : Integer;

begin
  for i := 1 to 20 do
  begin
    rand := Random(max);
    WriteLn('rand = ', rand);
  end;
end.
