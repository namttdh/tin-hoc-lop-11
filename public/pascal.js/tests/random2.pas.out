rand = 0
rand = 6118
rand = 1895
rand = 7186
rand = 7366
rand = 996
rand = 7689
rand = 1021
rand = 7612
rand = 8346
rand = 6873
rand = 5521
rand = 8121
rand = 8729
rand = 3485
rand = 704
rand = 9822
rand = 9231
rand = 8462
rand = 3553
